src/query/ast/tests/it/testdata/expr-error.txt

```diff
@@ -2,12 +2,11 @@
 5 * (a and ) 1
 ---------- Output ---------
 error: 
-  --> SQL:1:8
+  --> SQL:1:3
   |
 1 | 5 * (a and ) 1
```

Comment:
is it the expected span?

Reply:
It's expected after fixing a bug, but the error report is not perfect.